backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -702,4 +706,48 @@ private UserInfo consolidateUser(
 
     return new UserInfo(apiUser, Optional.of(orgRoles), isSiteAdmin, userStatus);
   }
+
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public List<String> updateUserPrivilegesAndGroupAccess(
+      String username, String orgExternalId, boolean allFacilitiesAccess, OrganizationRole role) {
+    return updateUserPrivilegesAndGroupAccess(
+        username, orgExternalId, allFacilitiesAccess, List.of(), role);
+  }
+
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public List<String> updateUserPrivilegesAndGroupAccess(
+      String username,
+      String orgExternalId,
+      boolean allFacilitiesAccess,
+      List<UUID> facilities,
+      OrganizationRole role)
+      throws IllegalGraphqlArgumentException {
+
+    if (!allFacilitiesAccess && facilities.isEmpty()) {
+      throw new IllegalArgumentException(MOVE_USER_ARGUMENT_ERROR);
+    }
+
+    Organization newOrg = _orgService.getOrganization(orgExternalId);
```

Comment:
should we do a check that orgExternalId is not an empty string?

Reply:
Sorry to jump in here Bob, but I was curious when I saw this question. I don't think it's necessary since it'll already throw an error if the organization is not found
https://github.com/CDCgov/prime-simplereport/blob/d4404d89fd310a99382dbf7b2f50c366aae40a56/backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java#L126-L132